src/manifest/pixiManifest.ts

```diff
@@ -118,14 +122,17 @@ function collectAssets(
     }
 
     const bundleAssets = localBundle.assets;
-
     const finalAssets = asset.getFinalTransformedChildren();
 
-    if (asset.transformChildren.length > 0)
+    if (asset.transformChildren.length > 0 && !asset.inheritedMetaData[options.tags!.mIgnore!])
     {
+        const nonIgnored = finalAssets.filter((finalAsset) => !finalAsset.inheritedMetaData[options.tags!.mIgnore!]);
```

Comment:
lets rename to something like `finalManifestAssets` or something along those lines!